Makefile

```diff
@@ -91,7 +91,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1 ;\
+	GO111MODULE=on go get github.com/minio/controller-tools/cmd/controller-gen@v0.4.6 ;\
```

Comment:
This seems to be a fork, can we go with the vanilla upstream @ https://github.com/kubernetes-sigs/controller-tools/releases?